cypress/integration/org_campaign_page.spec.ts

```diff
@@ -0,0 +1,23 @@
+describe('/o/[orgId]/campaigns/[campId]', () => {
+    beforeEach(() => {
+        cy.intercept('GET', /\/api\/orgs\/1\/campaigns\/941\/actions$/, { fixture: 'dummyEvents' });
+        cy.intercept('GET', /\/api\/orgs\/1\/campaigns\/941$/, { fixture: 'dummyCampaign' });
+    });
+
+        it('contains a campaign title', () => {
+            cy.visit('/o/1/campaigns/941');
+            cy.get('[data-test="campaign-heading"]').should('be.visible');
```

Comment:
This works, but it doesn't actually check that the title is being rendered, just that an element (which may or may not contain the title) with a certain ID is being rendered.

I would have preferred checking for the actual title from `dummyCampaign`.